<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user</class>
 <widget class="QWidget" name="user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_user">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>416</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>存款</string>
    </attribute>
    <widget class="QLabel" name="label_saving_bankcard_number">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>银行卡号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_saving_amount_money">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>存款金额：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_saving_bankcard_number">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_saving_amount_money">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_saving_ok">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>290</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_saving_back">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>取款</string>
    </attribute>
    <widget class="QLabel" name="label_draw_bankcard_number">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>银行卡号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_draw_take_money">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>150</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>取款金额：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_draw_take_money">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_draw_ok">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>75</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_draw_back">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>270</y>
       <width>75</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>转款</string>
    </attribute>
    <widget class="QLabel" name="label_change_bankcard_number">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>银行卡号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_other_bankcard_number">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>转向卡号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_change_transfer_money">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>转账金额：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_change_bankcard_number">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_other_bankcard_number">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_change_transfer_money">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_change_ok">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>75</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_change_back">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>75</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>交易明细</string>
    </attribute>
    <widget class="QLabel" name="label_trade_bankcard_number">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>银行卡号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_trade_balance">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>可用余额：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_trade_balance">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>余额</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_trade_detail">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>310</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>明细</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_trade_back">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>310</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_trade_text">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>311</width>
       <height>151</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_detail_card">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>39</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>245</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
